app/services/users/profile.rb

```diff
@@ -163,9 +163,14 @@ def update_status_and_errors
 
     def facility(facility_id)
       cerner_facility_ids = user.va_profile.cerner_facility_ids || []
+
+      is_cerner = cerner_facility_ids.include?(facility_id)
+      is_cerner = true if facility_id == '668' && Flipper.enabled?(:cerner_override_668)
```

Comment:
@patrickvinograd could you let me know if you have any suggestions here? 🙂

Reply:
I really don't know how to write Ruby very well but I am wondering if we can rely on a standard naming convention here to do a dynamic lookup of facility IDs in the Cerner overrides. For example, instead of 

```
is_cerner = true if facility_id == '668' && Flipper.enabled?(:cerner_override_668)
is_cerner = true if facility_id == '757' && Flipper.enabled?(:cerner_override_757)
```

We do

```
is_cerner = cerner_facility_ids.include?(facility_id) || Flipper.enabled?("cerner_override_#{facility_id}")
```

That way we can add new Cerner overrides without needing to update the logic

Reply:
Updated! 🙂